bundles/org.openhab.binding.freebox/src/main/java/org/openhab/binding/freebox/internal/handler/FreeboxHandler.java

```diff
@@ -293,6 +306,10 @@ public FreeboxApiManager getApiManager() {
         return apiManager;
     }
 
+    public String getAppToken() {
+        return getConfigAs(FreeboxServerConfiguration.class).appToken;
```

Comment:
Can you use the cached config instead?
```suggestion
        return configuration.appToken;
```

Reply:
There was no global variable `configuration` but I added it.